electrum/util.py

```diff
@@ -1141,13 +1142,25 @@ def write_json_file(path, data):
         raise FileExportFailed(e)
 
 
+def os_chmod(path, mode):
+    """chmod aware of tmpfs"""
+    try:
+        os.chmod(path, mode)
+    except OSError as e:
+        xdg_runtime_dir = os.environ.get("XDG_RUNTIME_DIR", None)
+        if xdg_runtime_dir and xdg_runtime_dir in path:
```

Comment:
The string contains/"in" keyword does not look robust.

maybe something like this instead?
`standardize_path(path).startswith(standardize_path(xdg_runtime_dir)+os.sep)`

maybe it might be worth creating a separate helper function for this, e.g. `is_subpath`